en/docs/services/create-your-first-service.md

```diff
@@ -134,14 +134,13 @@ import ballerinax/covid19;
 import ballerina/http;
 
 service / on new http:Listener(8090) {
-    resource function get stats/[string country](http:Caller caller, http:Request request) returns error? {
+    resource function get stats/[string country](http:Caller caller) returns error? {
 
         covid19:Client covid19Client = check new ();
-
         covid19:CovidCountry statusByCountry = check covid19Client->getStatusByCountry(country);
         var totalCases = statusByCountry?.cases ?: 0d;
-        worldbank:Client worldBankClient = check new ();
-        worldbank:CountryPopulation[] populationByCountry = check worldBankClient->getPopulationByCountry(country);
+        worldbank:Client worldbankEndpoint = check new ();
```

Comment:
Why u change this? Text says to use `worldBankClient` as the connection name.

Reply:
Corrected.